<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showAnCalChrRes</class>
 <widget class="QDialog" name="showAnCalChrRes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysis results</string>
  </property>
  <widget class="QGraphicsView" name="canvasCroped">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>240</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>3</y>
      <width>386</width>
      <height>142</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QCheckBox" name="chbBlue">
       <property name="text">
        <string>Blue</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pbSaveAnalysis">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save parameters into XML&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="lstImgs.qrc">
         <normaloff>:/new/icons/imagenInte/add-484.png</normaloff>:/new/icons/imagenInte/add-484.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pbClearCalib">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clear all calibration settings&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="lstImgs.qrc">
         <normaloff>:/new/icons/imagenInte/edit-clear.png</normaloff>:/new/icons/imagenInte/edit-clear.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="chbGreen">
       <property name="text">
        <string>Green</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="chbRed">
       <property name="text">
        <string>Red</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <widget class="QLineEdit" name="txtQuadFilename"/>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Quadrant file-name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="chbRedLine">
       <property name="text">
        <string>Red line</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="chbGreenLine">
       <property name="text">
        <string>Green line</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="chbBlueLine">
       <property name="text">
        <string>Blue line</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pbSaveScene">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save Graphic-view&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="lstImgs.qrc">
         <normaloff>:/new/icons/imagenInte/guardar.gif</normaloff>:/new/icons/imagenInte/guardar.gif</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pbInFolder">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open in folder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="lstImgs.qrc">
         <normaloff>:/new/icons/imagenInte/folder-icon.png</normaloff>:/new/icons/imagenInte/folder-icon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbCloseThis">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="lstImgs.qrc">
     <normaloff>:/new/icons/imagenInte/application-exit.png</normaloff>:/new/icons/imagenInte/application-exit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="lstImgs.qrc"/>
 </resources>
 <connections/>
</ui>
